3p/ampcontext.js

```diff
@@ -312,8 +313,20 @@ export class AbstractAmpContext {
       this.setupMetadata_(this.win_.name);
     }
   }
-}
 
+  /**
+   * Send 3p error to parent iframe
+   */
+  report3pError() {
+    this.win_.addEventListener('error', event => {
```

Comment:
This needs to be `this.win_.onerror = …` for legacy reasons.

Note, the very magic signature which only in some browsers comes with an error event.